docs/LogGuide.md

```diff
@@ -925,22 +1492,49 @@ To control aspects of the user interface, the game sends packets called Actor Co
 
 Actor control commands are identified by a category, with parameters passed to it as a handler. DirectorUpdate is a category of ActorControlSelf and is used to control the events inside content for an individual player:
 
-* BGM change
-* some cutscenes
-* barrier up/down
-* fade in/out
+- BGM change
+- some cutscenes
+- barrier up/down
+- fade in/out
 
-Structure:
-`21:TypeAndInstanceContentId:Command (4 bytes):Data (4x 4? byte extra data)`
+<!-- AUTO-GENERATED-CONTENT:START (logLines:type=ActorControl&lang=en) -->
 
-Examples:
+#### Structure
+
+```log
+Network Log Line Structure:
+33|2021-04-26T14:11:35.0000000-04:00|[instance]|[command]|[data0]|[data1]|[data2]|[data3]
+
+ACT Log Line Structure:
+[14:11:35.0] 21:33:2021-04-26T14:11:35.0000000-04:00:[instance]:[command]:[data0]:[data1]:[data2]:[data3]
+```
+
+#### Regexes
+
+```log
+Network Log Line Regex:
+^(?<type>(?:33))\|(?<timestamp>(?:[^|]*))\|(?<instance>(?:[^|]*))\|(?<command>(?:[^|]*))\|(?<data0>(?:[^|]*))\|(?<data1>(?:[^|]*))\|(?<data2>(?:[^|]*))\|(?<data3>(?:[^|]*))\|
+
+ACT Log Line Regex:
+(?<timestamp>(?:^.{14})) 21:(?<instance>(?:.*?)):(?<command>(?:.*?)):(?<data0>(?:.*?)):(?<data1>(?:.*?)):(?<data2>(?:.*?)):(?<data3>(?:.*?))$
+```
+
+#### Examples
 
 ```log
-21:8003753A:40000010:00:00:00:00
-21:80034E52:8000000D:1601:00:00:00
-21:80037543:80000004:257:00:00:00
+Network Log Line Examples:
+33|2021-04-26T14:17:16.7440000-04:00|80034E5B|8000000C|16|E74|00|00|55e425f79b5cadd724dbc718b86f70b1
```

Comment:
I think it'd be good to keep the 40000010 wipe line as an example since that's used in a lot of places.